calc/src/mechanics/gen78.ts

```diff
@@ -388,7 +415,7 @@ export function calculateSMSS(
   // #region (Special) Attack
   const attack = calculateAttackSMSS(gen, attacker, defender, move, field, desc, isCritical);
   const attackSource = move.named('Foul Play') ? defender : attacker;
-  if (move.named('Photon Geyser', 'Light That Burns The Sky')) {
+  if (move.named('Photon Geyser', 'Light That Burns The Sky', 'Tera Blast')) {
```

Comment:
tera blast only does the category adjustment if the user is terastallized

Reply:
Whoah, I didn't actually know that.  Thanks for the catch!